src/app/admin/bangumi-detail/video-file-modal/video-file-modal.component.ts

```diff
@@ -38,6 +38,8 @@ export class VideoFileModal implements OnInit, OnDestroy {
         DOWNLOADED: VideoFile.STATUS_DOWNLOADED
     };
 
+    enable_blob_storage_options = false;
```

Comment:
please consider using camelCase style variable/field name, the underscore style is a legacy issue and it's already defined in API which cannot be changed easily. But this field is not related to the API.